Controller/ExceptionController.php

```diff
@@ -76,7 +75,7 @@ public function showAction(Request $request, $exception, DebugLoggerInterface $l
             ));
         }
 
-        $format = $this->getFormat($request, $format);
+        $format = $this->getFormat($request, $request->getRequestFormat());
```

Comment:
actually I am wondering if we can do anything smarter here .. obviously we cannot refactor ``getFormat()`` too much without a BC break but obviously now a bunch of logic in the method is now sort of superfluous by default.